src/clients/Transmission.ts

```diff
@@ -27,10 +31,22 @@ interface TorrentMetadata {
 	name: string;
 }
 
+enum TorrentStatus {
+	Stopped = 0,
+	VerificationQueued,
+	Verifying,
+	DownloadQueued,
+	Downloading,
+	SeedingQueued,
+	Seeding,
```

Comment:
can you add a link in a comment so we can verify these enum values are in the correct order?

Reply:
The torrent statuses are described in this section of the RPC spec, a bit further in the paragraph (search for `A number between 0 and 6, where`):
https://github.com/transmission/transmission/blob/main/docs/rpc-spec.md#33-torrent-accessor-torrent-get